.cirrus.yml

```diff
@@ -276,6 +276,20 @@ ubuntu24_task:
     << : *RESOURCES_TEMPLATE
   << : *CI_TEMPLATE
 
+# Same as above, but running the ZAM tests instead of the regular tests.
+ubuntu24_zam_task:
+  container:
+    # Ubuntu 24.04 EOL: Jun 2029
+    dockerfile: ci/ubuntu-24.04/Dockerfile
+    << : *RESOURCES_TEMPLATE
+  << : *CI_TEMPLATE
+  env:
+    ZEEK_CI_SKIP_UNIT_TESTS: 1
+    ZEEK_CI_SKIP_EXTERNAL_BTESTS: 1
+    ZEEK_CI_BTEST_EXTRA_ARGS: -a zam
+    # Use a lower number of jobs due to OOM issues with ZAM tasks
```

Comment:
Without ASAN this might not be needed. Maybe remove before merge and see how it behaves?